includes/Data/ContentBlocksResolver.php

```diff
@@ -173,6 +170,10 @@ private static function is_block_empty( array $block ): bool {
 			return false;
 		}
 
+		if ( isset( $block['blockName'] ) && 'core/synced-pattern' === $block['blockName'] ) {
```

Comment:
Probably more holistic to check for a non-empty `ref` or `slug` attribute instead of looking for our custom `core/synced-pattern` shim